commands.txt

```diff
@@ -1,4 +1,20 @@
-if [ `uname` = 'Darwin' ]; then export SD_CURL_CMD_WRAPPER="eval"; else export SD_CURL_CMD_WRAPPER="$(echo 'sd-step exec --pkg-version 7.54.1 core/curl')"; fi
+if [ `uname` = 'Darwin' ]; then
+    export SD_CURL_CMD_WRAPPER="eval"
+    export SD_ZIP_CMD_WRAPPER=""
+else
```

Comment:
This probably won't work, inside our code, we concat each newline with `&&`.
https://github.com/screwdriver-cd/artifact-bookend/blob/master/index.js#L42

Reply:
fixed